src/engine/qcommon/common.cpp

```diff
@@ -52,25 +52,27 @@ Maryland 20850 USA.
 #include "sys/sys_events.h"
 #include <common/FileSystem.h>
 
+Cvar::Cvar<bool> com_developer = Cvar::Cvar<bool>( "developer", "", Cvar::TEMPORARY, false );
+Cvar::Cvar<int> com_speeds = Cvar::Cvar<int>( "com_speeds", "", Cvar::NONE, 0 );
+Cvar::Cvar<float> com_timescale( "timescale", "", Cvar::CHEAT | Cvar::SYSTEMINFO, 1.0f );
+
 cvar_t *com_pid; // bani - process id
 
-cvar_t *com_speeds;
-cvar_t *com_developer;
-cvar_t *com_timescale;
-cvar_t *com_dropsim; // 0.0 to 1.0, simulated packet drops
+Cvar::Range<Cvar::Cvar<float>> com_dropsim = Cvar::Range<Cvar::Cvar<float>>( "com_dropsim", "simulated packet drop", Cvar::CHEAT, 0.0f, 0.0f, 1.0f );
```

Comment:
I think you can remove the ` = Cvar::Cvar<…>` part of this line

Reply:
nitpick.